src/sentry/api/serializers/models/organization.py

```diff
@@ -59,6 +60,35 @@
     from sentry.api.serializers import UserSerializerResponse, UserSerializerResponseSelf
 
 
+class BaseOrganizationSerializer(serializers.Serializer):  # type: ignore
+    name = serializers.CharField(max_length=64)
+    slug = serializers.RegexField(r"^[a-zA-Z0-9][a-zA-Z0-9-]*(?<!-)$", max_length=50)
+
+    def validate_slug(self, value):
```

Comment:
im not too sure but i think the backend-typing check isn't happy about this not being typed